go.mod

```diff
@@ -75,3 +79,5 @@ replace gopkg.in/yaml.v2 => gopkg.in/yaml.v2 v2.2.2
 replace honnef.co/go/tools => honnef.co/go/tools v0.0.1-2019.2.3
 
 replace sigs.k8s.io/yaml => sigs.k8s.io/yaml v1.1.0
+
+replace knative.dev/test-infra => knative.dev/test-infra v0.0.0-20200610004422-8b4a5283a123
```

Comment:
Pin to specific knative.dev/test-infra to fix the error when running `./hack/update-deps.sh --upgrade`
The error logs is as follows:

`github.com/google/knative-gcp/hack imports
	knative.dev/pkg/hack imports
	knative.dev/test-infra/scripts: ambiguous import: found package knative.dev/test-infra/scripts in multiple modules:
	knative.dev/test-infra v0.0.0-20200608183332-3442736aebd0 (/usr/local/google/home/xiyue/go/pkg/mod/knative.dev/test-infra@v0.0.0-20200608183332-3442736aebd0/scripts)
	knative.dev/test-infra/scripts v0.0.0-00010101000000-000000000000 (/usr/local/google/home/xiyue/go/pkg/mod/knative.dev/test-infra/scripts@v0.0.0-20200`

Reply:
Not sure whether it's the right way to handle this(I searched Google and that's what stackoverflow suggested)

Reply:
can you add a comment saying that you are getting an ambiguous import problem otherwise?... in package knative.dev/test-infra/scripts

@chizhg any ideas?

Reply:
Added comments

Reply:
It looks like the ambiguous import is because `knative serving` also specifies specific version of `knative.dev/test-infra`

Reply:
@chizhg  fixed it here: https://github.com/knative/test-infra/pull/2185
can you revert this test-infra portion once the other PR gets merged and update deps again?